pyccel/ast/type_annotations.py

```diff
@@ -128,6 +128,14 @@ def is_const(self):
         """
         return self._is_const
 
+    @is_const.setter
+    def is_const(self, val):
+        self._is_const = val
+
+    @order.setter
+    def order(self, order):
+        self._order = order
```

Comment:
If you do not mind, I would move this setter to just after the corresponding getter.

By the way, it could be useful to check that `order in ['C', 'F']`.

Reply:
Done 15c24c13